Latest version will not compile

I can't seem to compile this latest version. When I run this:

go get github.com/mesosphere/mesos-dns

I get this error:

src/github.com/mesosphere/mesos-dns/main.go:85: syntax error: unexpected range, expecting {
src/github.com/mesosphere/mesos-dns/main.go:89: syntax error: argument to go/defer must be function call
src/github.com/mesosphere/mesos-dns/main.go:92: non-declaration statement outside function body
src/github.com/mesosphere/mesos-dns/main.go:105: non-declaration statement outside function body
src/github.com/mesosphere/mesos-dns/main.go:106: syntax error: unexpected }

Thanks,
Tim

Okay, tried that but updatedeps got this:

[vagrant@node1 mesos-dns]$ make updatedeps
--> Updating build dependencies
godep: not in manifest: .
godep: no packages can be updated
make: *** [updatedeps] Error 1

--- a/main.go
+++ b/main.go
@@ -82,7 +82,7 @@ func main() {
 		defer util.HandleCrash()
 		reloadTimeout := time.Second * time.Duration(config.RefreshSeconds)
 		reloadTimer := time.AfterFunc(reloadTimeout, tryReload)
-		for range reloadSignal {
+		for _ = range reloadSignal {
 			resolver.Reload()
 			logging.PrintCurLog()
 			reloadTimer.Reset(reloadTimeout)

